content/service_dialogs/transform-vm.yml

```diff
@@ -28,7 +28,7 @@
         required: false
         required_method:
         required_method_options: {}
-        default_value: ''
+        default_value: 'migration_group'
```

Comment:
@smelamud Could you explain what does this value represent ?

Reply:
@masayag This is a special tag category that is meant for tagging VMs for mass migration. See https://github.com/ManageIQ/manageiq/pull/16402

Reply:
Thanks for clarifying, we'll wait for https://github.com/ManageIQ/manageiq/pull/16402 to be merged first.